main.py

```diff
@@ -1,5 +1,13 @@
 
 from trafficcloud import api
+import sys
 
 if __name__ == "__main__":
-    api.runTrajectoryAnalysis('OlinShorter')
+    if len(sys.argv) <= 1:
+        print('Please pass the name of the folder in project_dir that you would like to run analysis on.')
+    else:
+        project = sys.argv[1]
+        if 'project_dir/' in project:
```

Comment:
Okay now from a usability standpoint, it looks like almost 99% of the time I'm going to be passing a relative path (since I'm tabcompleting to the project_identifier directory), and never an absolute path.  Your current code might be more readable in that case.  So really your decision.